v2/mqtt-to-pubsub/src/main/java/com/google/cloud/teleport/v2/templates/MqttToPubsub.java

```diff
@@ -80,6 +81,7 @@ public static void validate(MqttToPubsubOptions options) {
   }
 
   public static PipelineResult run(MqttToPubsubOptions options) {
+    Duration tenHours = Duration.standardHours(10);
```

Comment:
should this be one option for users to specify?

Reply:
I set it to max as a default, but I could change it so users specify it

Reply:
Note that with this change maxReadTime is still not user configurable. If I understood correctly the goal is to prevent the pipeline / some data read stuck indefinitely so sets a large and finite timeout, which user usually does not need to config.